### ISSUE TITLE
Allow multiple breakpoints for sidebars

### ISSUE THREAD

**Issue by stevenroussey-desiclabs** [framework7io/framework7#3192]
For Electron (and web), I would like the (left) sidebar to be a column of (large) icons. When hovering over the panel, it should expand (cover) to see the names related to the icons.

**Comment by nolimits4web** [comment]
I guess, you mean something like this https://framework7io.github.io/framework7-recipes/admin-dashboard/ ?

I think, it makes sense to add it to the core

**Comment by stevenroussey-desiclabs** [comment]
Yes, exactly! Also, nice option on desktop/electron when hovering over icon to have it expand (cover) in case we want to always have it as an icon view.

**Comment by nolimits4web** [comment]
This issue was incorrectly closed on 2026-05-19 by a compromised credential (Shai-Hulud npm supply-chain worm). Reopening — apologies for the noise.
